### PR TITLE
Gave 15 namespaces meta-labels

### PR DESCRIPTION
**Pull request by bryanmontalvan**
## Description
This commit is meant to add meta-labels to 15 namespaces for the continuation of adding labeled namespaces to the *Workload Overview dashboard*

## Note
Let me know if any docs or issues are incorrect, some I did leave empty due to not finding their docs or respective issues

### CODE REVIEW

File: cluster-scope/base/core/namespaces/debezium-ui/namespace.yaml [hunk 1]
```diff
@@ -5,3 +5,6 @@ metadata:
     annotations:
         openshift.io/display-name: Debezium UI
         openshift.io/requester: debezium
+        op1st/project-owner:
+        op1st/onboarding-issue:
```
Comment (dystewart) [inline cluster-scope/base/core/namespaces/debezium-ui/namespace.yaml:9]: I believe this is the onboarding issue: https://github.com/operate-first/support/issues/304

File: cluster-scope/base/core/namespaces/ds-ml-workflows-ws/namespace.yaml [hunk 2]
```diff
@@ -6,5 +6,8 @@ metadata:
     openshift.io/display-name: "Data Science: Machine Learning Workflows Workshop"
     openshift.io/requester: data-science
     operate-first.cloud/issue: "https://github.com/operate-first/support/issues/107"
```
Comment (dystewart) [inline cluster-scope/base/core/namespaces/ds-ml-workflows-ws/namespace.yaml:8]: Duplicate field on line 10?

File: cluster-scope/base/core/namespaces/ds-example-project/namespace.yaml [hunk 3]
```diff
@@ -6,5 +6,8 @@ metadata:
     openshift.io/display-name: "Data Science: Example project"
     openshift.io/requester: data-science
     operate-first.cloud/issue: "https://github.com/operate-first/support/issues/76"
```
Comment (dystewart) [inline cluster-scope/base/core/namespaces/ds-example-project/namespace.yaml:8]: Duplicate field on line 10?

File: cluster-scope/base/core/namespaces/ds-black-flake/namespace.yaml [hunk 4]
```diff
@@ -6,5 +6,8 @@ metadata:
     openshift.io/display-name: "Data Science: Black Flake"
     openshift.io/requester: data-science
     operate-first.cloud/issue: "https://github.com/operate-first/support/issues/76"
```
Comment (dystewart) [inline cluster-scope/base/core/namespaces/ds-black-flake/namespace.yaml:8]: Do we need both annotations here (lines 8 and 10)?
I noticed this in a couple other files so I''l just denote with: Duplicate field on line 10?

File: cluster-scope/base/core/namespaces/ds-example-project/namespace.yaml (hunk 3 above)
Comment (bryanmontalvan) [reply cluster-scope/base/core/namespaces/ds-example-project/namespace.yaml:8]: I'm a bit unsure about this myself, @4n4nd what do you think?

File: cluster-scope/base/core/namespaces/dex/namespace.yaml [hunk 5]
```diff
@@ -5,5 +5,8 @@ metadata:
   annotations:
     openshift.io/display-name: "Dex"
     openshift.io/requester: operate-first
+    op1st/project-owner: operate-first
+    op1st/onboarding-issue: "https://github.com/operate-first/apps/issues/65"
+    op1st/docs: "https://github.com/operate-first/apps/pull/248"
```
Comment (4n4nd) [inline cluster-scope/base/core/namespaces/dex/namespace.yaml:10]: docs: https://github.com/dexidp/dex

File: cluster-scope/base/core/namespaces/ds-example-project/namespace.yaml (hunk 3 above)
Comment (4n4nd) [reply cluster-scope/base/core/namespaces/ds-example-project/namespace.yaml:8]: the original label might be being used somewhere else, let's just add this one

File: cluster-scope/base/core/namespaces/ds-black-flake/namespace.yaml [hunk 6]
```diff
@@ -6,5 +6,8 @@ metadata:
...
+    op1st/project-owner: data-science
+    op1st/onboarding-issue: "https://github.com/operate-first/support/issues/76"
+    op1st/docs: "https://github.com/operate-first/apps/pull/205"
```
[the remark this answers is not in the document]
Comment (4n4nd) [reply cluster-scope/base/core/namespaces/ds-black-flake/namespace.yaml:11]: this namespace can be removed (@MichaelClifford )

**Review by 4n4nd — CHANGES_REQUESTED** [review]

File: cluster-scope/base/core/namespaces/examples-openshift-pub/namespace.yaml [hunk 7]
```diff
@@ -4,3 +4,6 @@ metadata:
     name: examples-openshift-pub
     annotations:
         openshift.io/requester: rbo
+        op1st/project-owner: examples-openshift-pub
```
Comment (4n4nd) [inline cluster-scope/base/core/namespaces/examples-openshift-pub/namespace.yaml:7]: This is not a valid Openshift group in [this list](https://github.com/operate-first/apps/tree/master/cluster-scope/base/user.openshift.io/groups).
@rbo Is there a group that should have access/ownership to this project?

File: cluster-scope/base/core/namespaces/ds-ml-workflows-ws/namespace.yaml (hunk 2 above)
Comment (4n4nd) [reply cluster-scope/base/core/namespaces/ds-ml-workflows-ws/namespace.yaml:8]: should be fine

Comment (4n4nd) [reply cluster-scope/base/core/namespaces/ds-ml-workflows-ws/namespace.yaml:8]: should be fine

File: cluster-scope/base/core/namespaces/ds-black-flake/namespace.yaml (hunk 6 above)
[the remark this answers is not in the document]
Comment (bryanmontalvan) [reply cluster-scope/base/core/namespaces/ds-black-flake/namespace.yaml:11]: @4n4nd Are you saying I should remove this or are you asking Michael to remove it?

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Pull request merged**